hc/front/tests/test_add_check.py

```diff
@@ -3,12 +3,27 @@
 
 
 class AddCheckTestCase(BaseTestCase):
+    def setUp(self):
+        super(AddCheckTestCase, self).setUp()
+        self.url = "/checks/add/"
+        self.client.login(username="alice@example.org", password="password")
```

Comment:
I would recommend leaving this out of the setup so the login is part of the testCase.

Reply:
it would make the logic for the test flow more naturally.